mscore/editstafftype.cpp

```diff
@@ -485,12 +485,14 @@ void EditStaffType::tabStemThroughCompatibility(bool checked)
 void EditStaffType::updatePreview()
       {
       setFromDlg();
-      if (preview) {
+      if (preview && preview->score()) {
```

Comment:
That's what I had at first, but changed after discussion with @lasconic and @sidewayss. So I don't have a problem with changing back to my original proposal using a `Q_ASSERT()` rather than the `if ()`  ;-)